examples/aggregator/module/rust/src/aggregation.rs

```diff
@@ -0,0 +1,52 @@
+//
+// Copyright 2020 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+pub trait Monoid {
+    fn identity() -> Self;
+    fn combine(&self, other: &Self) -> Self;
+}
+
+pub struct Aggregation<T: Monoid> {
+    /// Aggregated data.
+    data: T,
+    /// Number of contributed data samples.
+    sample_number: u64,
+    /// The number of samples that should be collected before for revealing the aggregation.
```

Comment:
Clarify whether this inclusive or exclusive (either here on in the struct comment).